### ISSUE TITLE
How to Insert Text Between Subscribe Buttons

### ISSUE THREAD

**Issue by jbavington** [Simbul/baker#963]
Hello Crew!

Thanks for the awesome support forum.  Found many answers to many questions without having to bug anyone.

We have a newsstand app that I'm just putting the finishing touches on.  It has an auto-renewable subscription IAP.  My understanding is that we need to mention the particulars of this within the app itself, as per Apple's Schedule 2 Section 3.8 (see attached)

![apple schedule 2 section 3 8](https://f.cloud.github.com/assets/4700969/683881/70fa2c0c-d9e9-11e2-9dd7-ff2affb09061.png)

I'd like to put this text between the Subscribe button we have and the Restore purchases button we have.  e.g., "Take out an annual subscription and save!  At any point you may turn off auto-renew in your Apple account subscriptions page at least 24 hours before the auto-renew is due."

![text inserted between buttons](https://f.cloud.github.com/assets/4700969/683882/74a4cc0e-d9e9-11e2-91ae-47731e48d64e.png)

I'm not very familiar with Xcode so don't know how to insert text in the right spot (in ShelfViewController.m I assume)

Help appreciated!  Thank you.

Jason

**Comment by nin9creative** [comment]
You can't put this in the app description on iTunes?

Regards,
Andrew

**Comment by jbavington** [comment]
Hi Andrew,

I can put it there for sure.  But I've seen other newsstand apps, e.g., MacFormat, put it right beside the subscribe buttons within the app and I want to avoid any reasons for Apple to reject the Binary (and then having to start from the very beginning of the Review queue).

Thanks.

**Comment by Simbul** [comment]
The component you're referring to is a `UIActionSheet`. As far as I can see in the docs, the component can only contain a title ("You are not subscribed" in your screenshot) and buttons. There's just no way of putting all the text you need in that component.

However, have a look at #958: it seems like putting information in the application description on iTunes Connect will be enough to be approved.

**Comment by jbavington** [comment]
Hi Simbul,

Thanks for your response.  You're definitely right that it can only contain a title.  I got my programmer who was working on another app project to take a look a this issue and he rewrote the section and created a custom UIViewController and made it pop up.  The result was I was able to include that text.

Thanks for referring to issue #958 .  Yeah, looks like putting it in the description would be ok.  I thought I'd play it safe with the direct in-app approach, taking a cue from larger magazine apps that seem to be doing this.
